kernel/dev/serial/rs232.cpp

```diff
@@ -82,3 +90,14 @@ void px_rs232_init(uint16_t com_id) {
         "Panix Serial Output Debugger\n\n"
     );
 }
+
+px_ring_buff_t* px_rs232_init_buffer(int size) {
```

Comment:
Does this function ever get called? I don't seen any references to it.

Reply:
It's called in main. I didn't commit main because I didn't know if we actually wanted to use this yet since all it would do right now is print to screen. I figured I would wait until we had a real use for it.

Reply:
Ah okay, so this branch doesn't technically support input then?

Reply:
It contains all of the logic and functions necessary to read input. I tested it on my machine and the input successfully printed to the screen. It does not, however, enable that functionality because I wasn't 100% of how we wanted to use it right now and figured we could decide that later.

Reply:
Okay. I guess I just figured we'd enable it, and then let applications consume that information at a later time.

Reply:
I can do that. I'll make another commit later.

Reply:
Added to main.